client/src/components/Suggestions.jsx

```diff
@@ -18,13 +18,12 @@ class Suggestions extends React.Component {
   componentDidMount() {
     axios.get(`/restaurants/${this.props.id}/suggestions`)
       .then((response) => {
-        // const data = [];
-        // while (response.data.length) {
-        //   data.push(response.data.splice(0, 3));
-        // }
-        console.log(response.data);
+        const data = [];
+        for (let i = 0; i < response.data.length; i += 1) {
+          data.push(response.data[i][0]);
+        }
```

Comment:
If you're just iterating through all the elements of `response.data`, then using `map` might be cleaner.

Reply:
great catch! I will implement that